index.html

```diff
@@ -222,8 +235,63 @@ <h3>tccc mode</h3>
     <p>Overlapping more than two notes is undefined behavior in tccc mode. Don't do it.</p>
   </div>
 
+  <div id="pitchbend" class="container">
+    <h2>Pitch Bend Events</h2>
+
+    <p>TCCC can perform additional adjustments to the pitch of notes using MIDI Pitch Bend events, allowing you to 
+      set notes to microtonal values.</p>
+
+    <ul>
+      <li>In your DAW, MIDI pitch bend events are a numerical value that is converted to a change in pitch 
+        by the output device (e.g. your synthesizer plugin, soundfont player, etc).</li>
```

Comment:
I'd mildly prefer not to explain the basics of pitch events here, since users can just google it and get more in-depth answers than what we would have. (The stuff about how TCCC uses pitch bends is all good.)

Reply:
I agree with this and the previous comment about external sources, and did feel I could be overexplaining it. However one part I feel is important is make sure users know to set the TCCC pitch range to match their DAW setup, otherwise the pitch changes won't match.

Reply:
Updated tutorial to remove the pitch basics: 1c6745acfe353ba194f60a47e548da83353aeedf